addons/account/views/account_report.xml

```diff
@@ -30,7 +30,6 @@
             report_type="qweb-pdf"
             name="account.account_invoice_report_duplicate_main"
             file="account.report_invoice_duplicate"
-            attachment_use="False"
```

Comment:
The content of the newly generated report is always returned instead of returning the content of an existing attachment. Then, the attachment_use field became unused. Note this behavior has already been changed in master.